### PR TITLE
added top-section

### PR DESCRIPTION
**Pull request by kelsywells**
Only have my top section finished, without mixins or variables (because no need for them in top section it seems) but they will be added when my mid section is finished this evening!

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -1 +1,242 @@
...
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
```
Comment (ari7946) [inline css/index.css]: Do you think adding an empty line between selectors makes the code easier to read?

File: index.html [hunk 2]
```diff
@@ -4,16 +4,90 @@
...
+        If you hire me I will be a dedicated and loyal employee who 
+        will do what it takes to help and grow with the company.</p>
+    </section>
+
+  <section class="mid-section">
+
+    <section id="experience">
+      <h3>Experience</h3>
+
+      <p>Dilettante Mocha Cafe </br>
```
Comment (ari-mercado) [inline index.html:52]: This works. Another way to break lines is to wrap each piece in "p" tags. Additionally, you can set the line height of the "p" tags using the "line-height" property.

File: less/index.less [hunk 3]
```diff
@@ -54,6 +59,150 @@ table {
 }
```
Comment (ari-mercado) [inline less/index.less:60]: The code looks clean! My only suggestion is to make it mobile friendly (max-width 500px).

**Comment by ari7946** [comment]
Great job, Kelsy! The design and colors look amazing!

**Review by ari7946 — APPROVED** [review]

**Pull request merged**